changelogs/8.8.asciidoc

```diff
@@ -0,0 +1,26 @@
+[[release-notes-8.8]]
+== APM version 8.8
+
+https://github.com/elastic/apm-server/compare/8.7\...8.8[View commits]
+
+* <<release-notes-8.8.0>>
+
+[float]
+[[release-notes-8.8.0]]
+=== APM version 8.8.0
+
+[float]
+==== Breaking Changes
+- Store app logs into service-specific data streams{pull}10456[10456]
```

Comment:
```suggestion
- Store app logs into service-specific data streams {pull}10456[10456]
```
Without a space before the attribute it doesn't render correctly:
<img width="679" alt="Screenshot 2023-04-26 at 8 45 00 AM" src="https://user-images.githubusercontent.com/5618806/234629796-f9997b1e-e1d5-4bde-a070-5c7753c2278f.png">

Reply:
Actually, would you mind updating the 8.7 release notes as well? There's an error here:

https://github.com/elastic/apm-server/blob/a50b9a05bede9e7c3a2482ec2feb1e219e00c4c1/changelogs/8.7.asciidoc?plain=1#L84-L85

It should be:

```diff
-  - Improve APM UI query performance by producing `1m`, `10m` and `60m` aggregation intervals for service destination metrics. Store them into data streams `metrics-apm.service_destination.${interval}` and make them hidden for non-default intervals {pull} 
-  926[9926] {pull}10070[10070] 
+  - Improve APM UI query performance by producing `1m`, `10m` and `60m` aggregation intervals for service destination metrics. Store them into data streams `metrics-apm.service_destination.${interval}` and make them hidden for non-default intervals 
+  {pull}9926[9926] {pull}10070[10070] 
 ```

Reply:
Done!

Reply:
Thanks!